extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
And how do I what I asked in https://github.com/eclipse/smarthome/pull/1936#issuecomment-244682201?
"send whatever string it likes (without being constrained to a list of options)"?